provider.go

```diff
@@ -283,21 +277,13 @@ func (p *Provider) GetServicePrincipalToken(env *azure.Environment, resource str
 func (p *Provider) MountSecretsStoreObjectContent(ctx context.Context, attrib map[string]string, secrets map[string]string, targetPath string, permission os.FileMode) (err error) {
```

Comment:
May we add unit test for MountSecretsStoreObjectContent function to make sure we have successfully the returned object in the right format?